### DISCUSSION TITLE
The ref_path is ambiguous. As a result, the code redirection display is abnormal.

### DISCUSSION THREAD

**Discussion by feng9797** [community/community#128092] [Repositories] [answered]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

https://github.com/feng9797/0530
The current status of my repository, the main branch has a dev/dev directory, after creating a tag main/dev, the current page can not open the dev folder under the main branch, it will jump to the main/dev tag

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment, marked as answer]
Thanks for posting in the GitHub Community, @feng9797 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by feng9797** [reply]
Thank you. There are so many categories, it's really hard to tell them apart.
